index.js

```diff
@@ -27,6 +27,44 @@ exports.NativeScriptJsonpPlugin.prototype.apply = function (compiler) {
     });
 };
 
+exports.ExcludeUnusedElements = function () {
+    //
+}
+
+exports.ExcludeUnusedElements.prototype = {
+    apply: function (compiler) {
```

Comment:
Why not just `exports.ExcludeUnusedElementsPlugin.prototype.apply = function...` ?